src/Tests/FileEntityCreationTest.php

```diff
@@ -73,6 +73,8 @@ public function testFileEntityCreationMultipleSteps() {
     // Create a file.
     $edit = array();
     $edit['files[upload]'] = drupal_realpath($test_file->uri);
+    $this->drupalGet('file/add');
+    $this->assertFalse($this->xpath('//input[@id="edit-upload-remove-button"]'), 'Remove');
     $this->drupalPostForm('file/add', $edit, t('Next'));
```

Comment:
You can change this to NULL, as we already are on that page now. Otherwise we do a second request.